src/models/spaces/populations/mouse-model/hawks.ts

```diff
@@ -39,15 +89,103 @@ export const hawkSpecies = new Species({
     new Trait({ name: "wings", default: 0 }),
     new Trait({ name: "mating desire bonus", default: -40 }),
     new Trait({ name: "hunger bonus", default: 100 }),
-    new Trait({ name: "is immortal", default: true })
+    new Trait({ name: "is immortal", default: true }),
+    new Trait({ name: "height", default: 10 })
   ],
+  imageProperties: {
+    rotate: true,
+    initialRotationDirection: -Math.PI / 2
+  },
   imageRules: [{
     name: "hawk",
     contexts: ["environment"],
     rules: [{
       image: {
         path: "assets/curriculum/mouse/populations/hawk.png",
-        scale: 0.3,
+        // Annoying that we can't simply make scale a computed property.
+        // Something for the next version of populations.js ...
+        scale: 0.18,
+        anchor: {
+          x: 0.5,
+          y: 0.2
+        }
+      },
+      useIf: (agent: Agent) => agent.get("height") <= 2
+    }, {
+      image: {
+        path: "assets/curriculum/mouse/populations/hawk.png",
+        scale: 0.185,
+        anchor: {
+          x: 0.5,
+          y: 0.2
+        }
+      },
+      useIf: (agent: Agent) => agent.get("height") <= 3
+    }, {
+      image: {
+        path: "assets/curriculum/mouse/populations/hawk.png",
+        scale: 0.19,
+        anchor: {
+          x: 0.5,
+          y: 0.2
+        }
+      },
+      useIf: (agent: Agent) => agent.get("height") <= 4
+    }, {
+      image: {
+        path: "assets/curriculum/mouse/populations/hawk.png",
+        scale: 0.195,
+        anchor: {
+          x: 0.5,
+          y: 0.2
+        }
+      },
+      useIf: (agent: Agent) => agent.get("height") <= 5
+    }, {
+      image: {
+        path: "assets/curriculum/mouse/populations/hawk.png",
+        scale: 0.21,
+        anchor: {
+          x: 0.5,
+          y: 0.2
+        }
+      },
+      useIf: (agent: Agent) => agent.get("height") <= 6
+    }, {
+      image: {
+        path: "assets/curriculum/mouse/populations/hawk.png",
+        scale: 0.215,
+        anchor: {
+          x: 0.5,
+          y: 0.2
+        }
+      },
+      useIf: (agent: Agent) => agent.get("height") <= 7
+    }, {
+      image: {
+        path: "assets/curriculum/mouse/populations/hawk.png",
+        scale: 0.22,
+        anchor: {
+          x: 0.5,
+          y: 0.2
+        }
+      },
+      useIf: (agent: Agent) => agent.get("height") <= 8
+    }, {
+      image: {
+        path: "assets/curriculum/mouse/populations/hawk.png",
+        scale: 0.225,
+        anchor: {
+          x: 0.5,
+          y: 0.2
+        }
+      },
```

Comment:
I like the swoop.  It still feels a little subtle (don't think I would notice it unless I looked for it).  I probably would have incremented by .01 instead of .005 to make it more pronounced, but probably not worth investigating/adjusting now.